tests/report_tests/timeout_tests.py

```diff
@@ -35,10 +39,27 @@ class NativeCmdTimeoutTest(StageTwoReportTest):
     install_plugins = ['timeout_test']
     sos_cmd = '-o timeout_test -vvv'
 
+    def test_correct_plugin_timeout(self):
```

Comment:
Worth to add to some other `fake_test` this?

    from sos.utilities import TIMEOUT_DEFAULT

        def test_correct_plugin_timeout(self):
            man = self.get_plugin_manifest('binary_test')
            self.assertEquals(man['timeout'], TIMEOUT_DEFAULT)

? Well, that would introduce some dependency among fake plugins that isnt ideal, so maybe have another `defaulttimeout_test` plugin to have also similar tests like here for a plugin without hardcoded timeout?

Reply:
I folded these types of tests into the existing ones by enabling the `host` plugin and checking its timeout setting where appropriate. I think that should suffice?